### PR TITLE
[test-only] fix creating drawio file

### PR DESCRIPTION
**Pull request by ScharfViktor**
ocis Ci is red again:

fix creating drawio file

https://drone.owncloud.com/owncloud/ocis/20983/68/10
https://drone.owncloud.com/owncloud/ocis/20982/68/10

### CODE REVIEW

File: tests/e2e/support/objects/app-files/share/collaborator.ts [hunk 1]
```diff
@@ -311,20 +311,19 @@ export default class Collaborator {
     const collaboratorRow = Collaborator.getCollaboratorUserOrGroupSelector(collaborator, type)
     await page.locator(collaboratorRow).waitFor()
 
-    await page
-      .locator(util.format(Collaborator.collaboratorEditDropdownButton, collaboratorRow))
-      .click()
-    await page
-      .locator(util.format(Collaborator.removeExpirationDateCollaboratorButton, collaboratorRow))
-      .click()
-
```
Comment (saw-jan) [inline tests/e2e/support/objects/app-files/share/collaborator.ts:320]:
```diff
+ await page
+   .locator(util.format(Collaborator.collaboratorEditDropdownButton, collaboratorRow))
+   .click()
```

File: tests/e2e/support/objects/app-files/share/collaborator.ts [hunk 2]
```diff
@@ -311,20 +311,19 @@ export default class Collaborator {
...
           resp.status() === 200 &&
           resp.request().method() === 'POST'
-      )
+      ),
+      page
+        .locator(util.format(Collaborator.collaboratorEditDropdownButton, collaboratorRow))
+        .click(),
+      page
+        .locator(util.format(Collaborator.removeExpirationDateCollaboratorButton, collaboratorRow))
+        .click()
```
Comment (saw-jan) [inline tests/e2e/support/objects/app-files/share/collaborator.ts:326]:
```suggestion
      ),
      page
        .locator(util.format(Collaborator.removeExpirationDateCollaboratorButton, collaboratorRow))
        .click()
```

**Review by saw-jan — COMMENTED** [review]
Opening dropdown first and then checking respose together with remove expiration action

File: tests/e2e/support/objects/app-files/share/collaborator.ts [hunk 3]
```diff
@@ -310,22 +310,22 @@ export default class Collaborator {
...
-
-    await Promise.all([
-      page.waitForResponse(
-        (resp) =>
-          resp.url().includes('shares') &&
-          resp.status() === 200 &&
-          resp.request().method() === 'POST'
-      )
-    ])
+      .click(),
```
Comment (SwikritiT) [inline tests/e2e/support/objects/app-files/share/collaborator.ts:315]:
```suggestion
      .click()
```

Comment (ScharfViktor) [reply tests/e2e/support/objects/app-files/share/collaborator.ts]: ah, sorry. Not awake yet.

**Review by SwikritiT — APPROVED** [review]
The linter is complaining, otherwise looks good to me

**Review by saw-jan — APPROVED** [review]

**Pull request merged**